packages/patternfly-4/react-core/src/layouts/Gallery/Gallery.test.tsx

```diff
@@ -1,11 +1,11 @@
-import React from 'react';
-import Gallery from './Gallery';
+import * as React from 'react';
+import { Gallery } from './Gallery';
 import { GutterSize } from '../../styles/gutters';
 import { shallow } from 'enzyme';
 
 Object.values(GutterSize).forEach(gutter => {
   test(`gutter ${gutter}`, () => {
-    const view = shallow(<Gallery gutter={gutter} />);
```

Comment:
This does not make sense to me.  You cant have `const view = shallow(<Gallery sm/>);` or `const view = shallow(<Gallery lg/>);` etc